### PR TITLE
[vscode] Add share command

### PR DESCRIPTION
**Pull request by rossdanlm**
[vscode] Add share command

Same functionality as from Gradio where we first upload to S3, and then pass in that source Uri to our aiconfig endpoint

Some things I still need to do:
1. Copy pasting feature --> right now it's kind of bulky and it just shows up tin eh dialogue at bottom right. Would be better UX to have it be copy-paste functionality
2. Remove hardcoding to file

Another thing I noticed is that when we try to open the HF extensions (or any extension that relies on non-core model parsers), the VSCode extension will fail. I think this is fine, just need to think more about the UX and how to globally register other model parsers to render this (also without relying on the client code to display input schema settings)
3. Remove hardcode and only show active open window

## Test Plan
Error gets properly thrown if s3 bucket is wrong

Screenshot 2024-01-31 at 00 20 18

Share URI works

https://github.com/lastmile-ai/aiconfig/assets/151060367/d1791c5c-e450-4ffa-a61e-e7af8488073b

### CODE REVIEW

File: vscode-extension/src/extension.ts
Comment (rholinshead) [inline vscode-extension/src/extension.ts:45]: Does this need any handling to work without aws credentials (as in https://github.com/lastmile-ai/gradio-workbook/pull/153/files)?

File: vscode-extension/src/extension.ts
Comment (rholinshead) [inline vscode-extension/src/extension.ts:37]: Should do this before landing

File: vscode-extension/src/extension.ts
Comment (rholinshead) [inline vscode-extension/src/extension.ts:70]: Will the share button also show its own modal on top of this?

Comment (rholinshead) [reply vscode-extension/src/extension.ts:70]: Oh, I guess this is unrelated to the button at all, just a command palette command. Nvm

File: vscode-extension/src/extension.ts
Comment (rossdanlm) [reply vscode-extension/src/extension.ts:37]: cc @saqadri can you land final version where you had it updated?

File: vscode-extension/src/extension.ts
Comment (rholinshead) [reply vscode-extension/src/extension.ts:45]: Oh, ok. Can you please update the comment to make that a bit clear (specifically noting 'without credentials')

**Review by rholinshead — APPROVED** [review]
Accepting to unblock, just the updated comment and the fix for the filename before landing

**Comment by saqadri** [comment]
This PR can be closed, right?

**Pull request closed without merging**
